lib/rental.rb

```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_accessor :date
+  attr_reader :person, :book
```

Comment:
- [ ] Since a rental belongs to both a person and a book instance, I'd advise defining both Getters and Setters for `@book` and `@person` instance variables, instead of just using a getter.